apps/backend/src/mcp/tools/device_management.py

```
@@ -26,6 +26,150 @@
 logger = logging.getLogger(__name__)
 
 
+async def add_device(
+    hostname: str,
+    device_type: str = "server",
+    description: Optional[str] = None,
```

Ruff (0.12.2)

Use `X | None` for type annotations

Convert to `X | None`

(UP045)